Mathlib/RingTheory/Ideal/Basic.lean

```diff
@@ -519,6 +519,24 @@ theorem span_singleton_eq_span_singleton {α : Type u} [CommRing α] [IsDomain 
   apply and_congr <;> rw [span_singleton_le_span_singleton]
 #align ideal.span_singleton_eq_span_singleton Ideal.span_singleton_eq_span_singleton
 
+/-- The equivalence between `Associates R` and the principal ideals of `R` defined by sending the
+class of `x` to the principal ideal generated by `x`. -/
+noncomputable def equivIsPrincipal (R : Type*) [CommRing R] [IsDomain R] :
```

Comment:
```suggestion
noncomputable def associatesEquivIsPrincipal (R : Type*) [CommRing R] [IsDomain R] :
```
Or maybe
```suggestion
noncomputable def _root_.Associates.equivIsPrincipal (R : Type*) [CommRing R] [IsDomain R] :
```
Same below

Reply:
I think I'd rather keep this def in the `Ideal` namespace, so I'll go for the first suggestion